src/libraries/System.Net.Security/tests/StressTests/SslStress/Readme.md

```diff
@@ -1,3 +1,56 @@
 ## SslStress
 
-Stress testing suite for SslStream
+Provides stress testing scenaria for System.Net.Security.SslStream.
```

Comment:
interesting: "scenaria". I thought the plural form of "scenario" was "scenarios". 

I wasn't aware of 'scenaria' until this PR.  :)

https://www.yourdictionary.com/scenaria

>"(nonstandard) plural form of scenario"

Reply:
Yeah, it's the greek/latin plural form. Also, did you know that the plural of "schema" is "schemata"? 😄